app/containers/Avatar/index.tsx

```diff
@@ -21,19 +25,24 @@ const AvatarContainer = ({
 	onPress,
 	getCustomEmoji,
 	isStatic,
-	rid
-}: IAvatar): React.ReactElement => {
+	rid,
+	handleEdit,
+	isUserProfile
+}: IAvatar & { isUserProfile?: boolean; handleEdit?: () => void }): React.ReactElement => {
```

Comment:
Don't you know from params and redux when the avatar is from the logged in user?

Reply:
It also creates an insane prop drilling